### ISSUE TITLE
Identification link does not work.

### ISSUE THREAD

**Issue by dashohoxha** [pparent76/Own-Mailbox_debian_install#8]
It still does not work, it sends me to: `http://127.0.0.1:8085/first/03b-Identification-cookie-wrong.html`

`torsocks wget google.fr` works.

`tail /var/log/syslog` has this line at the end:

```
Jun 19 21:52:53 cfb61fff46c5 postmulti[644]: fatal: /etc/mailname: cannot open file: \
    No such file or directory
```

Directory `/var/log/tor/` is empty.

Maybe you can ask some other friends to test it and see whether it is my problem, or it is a problem with own-mailbox.

**Comment by pparent76** [comment]
Very strange. Other people did test it and had no problem.

Can you execute:
 su www-data -s /bin/bash -c "torsocks wget YOUR_LINK -O /home/www-data/cookie" 

Ps: the /etc/mailname problem is normal.

**Comment by dashohoxha** [comment]
However, this fails:
`wget
https://proxy.omb.one:6565/request-omb/Create_Acounts/1mUGmthEOYfeUGjZBcuhqMmhJeAiF6uGspnPSEtG.cookie`

```
Resolving proxy.omb.one (proxy.omb.one)... failed: Name or service not
known.
wget: unable to resolve host address 'proxy.omb.one'
```

Also `dig proxy.omb.one` and `ping proxy.omb.one` fail.

On the host system however, all of these work.

**Comment by dashohoxha** [comment]
`/etc/resolv.conf` is `nameserver 127.0.0.1` which seems wrong to me, because I don't see and DNS server running.

**Comment by dashohoxha** [comment]
When I add `nameserver 8.8.8.8` on `/etc/resolv.conf` it works as expected.

**Comment by dashohoxha** [comment]
But then "Domain request failled for the following reason:" (no reason).

**Comment by pparent76** [comment]
Hi,

127.0.0.1 in /etc/resolv.conf is normal because you should have ttdnsd running, which is a dns resolver through tor. It is not normal that you can't do any DNS request.

Also your second wget and ping proxy.omb.one is not supposed to work on the real system with iptables, that blocks everything but tor.

If you are redirected to http://127.0.0.1:8085/first/03b-Identification-cookie-wrong.html . It means that the following command fails in the cgi-script:
torsocks wget $link -O /home/www-data/cookie > /tmp/download
It is very strange that changing /etc/resolv.conf makes any difference at this stage, because torsocks wget is not supposed to do DNS request.

I'll try to think of further tests you can do to see where the problem is coming from. It's possible that there are problems with the tor service. 

Thanks a lot for your time, it's very usefull!

Pierre.

**Comment by pparent76** [comment]
Could you give me the output of the following commands when you have time?

cat /var/lib/tor/omb_hidden_service/hostname
cat /tmp/download
cat /etc/tor/torrc
cat /home/www-data/cookie
ps -ae | grep tor
ps -ae | grep ttndsd
killall tor; fuser 9050/tcp; su tor -c tor&
killall ttdnsd; ttdnsd -l;

It should give me all the information I need to understand your problem.

Thanks a lot for your help and your time!

**Comment by dashohoxha** [comment]
Ok, but docker is supposed to avoid these kind of situations, where you send me the commands and I send you the output, and you try to guess what is going on. If you and I follow the same instructions for building the docker container, we should end up with exaclty the same result. So, you don't have to ask me what my system looks like, because yours is identic to mine. So, this kind of old-fashioned debuging makes no sense to me.
As a tester, I am supposed to run your instructions and verify whether it works as expected or not. Debugging and fixing is up to you.

**Comment by pparent76** [comment]
> Ok, but docker is supposed to avoid these kind of situations,

Ok I do agree, but I did compile the docker Image with the current version of git repository, on two different machines, on 2 different Internet connections, and everything works fine. And other testers did not report that problem.  So I don't see any other solution, to understand what happens. Your Internet connection, may have special properties, which can cause a problem.

But if you don't have the time to do it, or don't wish to do it, it does not mater at all! I do understand your time is limited and you already did a lot, I'm very grateful for that! ;)

Thanks!
Pierre.

**Comment by dashohoxha** [comment]
If it works for you, then it is fine.
I do like it as a project, and it is interesting, but I don't have an immediate need for having a private mailbox. So, I actually don't care if it does not work for me.

It is not a matter of time. I did not have time yesterday, but I did it. I do have plenty of time today, but I don't want to do it.

Anyway, this project is a great idea, I can only appraise you for it.
